Symfony/CS/Tests/Fixer/Symfony/UnaryOperatorsSpacesFixerTest.php

```diff
@@ -84,8 +84,8 @@ public function provideCases()
                 '<?php function foo(& $a, array & $b, Bar & $c) {}',
             ),
             array(
-                '<?php foo(+$a, -2,-$b, &$c);',
-                '<?php foo(+ $a, - 2,- $b, & $c);',
```

Comment:
I am thinking about some general list of exceptions for that assertion (not for Linter itself).